### PR TITLE
[#1] add db and migrations

### PR DESCRIPTION
**Pull request by wrutek**
- update gitignore
 - remove cached pyc file
 - add map model migration

### CODE REVIEW

**Comment by wrutek** [comment]
https://github.com/wrutek/hello_wars/issues/1

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
 db.sqlite3
-*pyc
+*.pyc
```
Comment (ikex) [inline .gitignore]: How about using [this boilerplate](https://github.com/github/gitignore/blob/master/Python.gitignore) or at least changing it to:

[](url)
```
# regular python
__pycache__/
*.py[cod]
*$py.class
# django
*.log
.static_storage/
.media/
local_settings.py
```

**Review by ikex — APPROVED** [review]
Except that my personal preference would be to use "columns" instead of "cols" I don't have any other comments to add.

I give 5 stars.

File: .gitignore (hunk 1 above)
Comment (wrutek) [reply .gitignore]: ok

**Pull request merged**